app/src/components/CustomPeriodDialog.tsx

```diff
@@ -58,6 +70,9 @@ export function CustomPeriodDialog(props: CustomPeriodDialogProps) {
             e.preventDefault();
             handleSelection();
           }}
+          onChange={() => {
```

Comment:
Could we make these validations on submit instead? Not a huge fan of disabling the confirm button while typing

Maybe by showing a red error message above the button

Reply:
@psikoi updated


https://github.com/wise-old-man/wise-old-man/assets/43260193/511edfd0-f806-490a-8ab1-d75210ee71c7